doc/nrf/releases_and_maturity/releases/release-notes-2.9.0-nRF54H20-1.rst

```diff
@@ -427,7 +427,7 @@ Zephyr
 
 .. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each nRF Connect SDK release.
 
-The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``beb733919d8d64a778a11bd5e7d5cbe5ae27b8ee``, with some |NCS| specific additions.
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``8d4a7b606b8298fe657a419a3c3609cce46ed119``, with some |NCS| specific additions.
```

Comment:
@bjarki-andreasen please review this and the changes right below

Reply:
@carlescufi Could you please review this?

Reply:
I don't think this is the case, beb733919d8d64a778a11bd5e7d5cbe5ae27b8ee is still the merge base from what I see, 8d4a7b606b8298fe657a419a3c3609cce46ed119 is a fromlist, so likely not to be the last upstream commit :)